ddterm/shell/sd_journal.js

```diff
@@ -75,7 +79,17 @@ export function sd_journal_stream_fd(identifier, priority = LOG_INFO, level_pref
     }
 
     try {
-        GLib.unix_set_fd_nonblocking(fd, false);
+        // BEGIN ESM
+        if (major > 45) {
+            const GLibUnix = Gi.require('GLibUnix', '2.0');
+            GLibUnix.set_fd_nonblocking(fd, false);
+        } else {
+        // END ESM
+        // eslint-disable-next-line indent
+            GLib.unix_set_fd_nonblocking(fd, false);
```

Comment:
Maybe `return fd` earlier and avoid `else {}`?

Reply:
Could you elaborate a little further?

Reply:
```suggestion
            GLibUnix.set_fd_nonblocking(fd, false);
            return fd;
        }
        // END ESM

        GLib.unix_set_fd_nonblocking(fd, false);
        return fd;
```

Reply:
Done :+1: